public/api/conf/1.0/schemas/subSchemas/accountingPeriod.json

```diff
@@ -4,12 +4,12 @@
     "startDate": {
       "description": "The start date of the accounting period",
       "type": "string",
-      "format": "date"
```

Comment:
I'd leave this as format date. Even though our validation library doesn't support it, it's better JSON Schema documentation